src/commcare_cloud/manage_commcare_cloud/monitors/cert_expire.yml

```diff
@@ -17,6 +17,6 @@ options:
   silenced: {}
   thresholds: {critical: 5, ok: 1, warning: 1}
   timeout_h: 0
-query: '"http.ssl_cert".over("*").exclude("environment:64-test","host:hqproxy0").by("host","instance","url").last(6).count_by_status()'
+query: '"http.ssl_cert".over("*").exclude("environment:64-test").by("host","instance","url").last(6).count_by_status()'
```

Comment:
I think this change will conflict with https://github.com/dimagi/commcare-cloud/pull/6361, so whichever is merged second will need to be have merge conflicts resolved.

Reply:
You are correct, of course. https://github.com/dimagi/commcare-cloud/pull/6365/commits/06006d199d704547cf6b938ad6b9558d6ebb08df